### PR TITLE
First release of Piedmontese language addon

### PR DESCRIPTION
**Pull request by glemco**
Added keyboard for the Piedmontese language (spoken in the Piedmont region in Italy).
I hope everything is fine for now, otherwise tell me!

### CODE REVIEW

File: addons/languages/piedmontese/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
Comment (martholomew) [inline addons/languages/piedmontese/apk/build.gradle:1]: Have this as false for now.

**Comment by glemco** [comment]
> Looks great! One thing, under `listings/$LANGUAGE/graphics` could you add a `phone-screenshots` folder and put some screenshots in there? It's formatted just like [this](https://github.com/AnySoftKeyboard/AnySoftKeyboard/tree/master/addons/languages/bulgarian/apk/src/main/play/listings/en-US/graphics/phone-screenshots).

Here you go, I added a new commit with the screenshots!
Thanks

File: addons/languages/piedmontese/apk/build.gradle (hunk 1 above)
Comment (menny) [reply addons/languages/piedmontese/apk/build.gradle:1]: @glemco please also set this to `false` for now. After reviewing we (the maintainers) we'll set this to `true`

**Comment by glemco** [comment]
Sure, I thought you would have done it!
Hope now it's fine

**Pull request merged**